uri/README.md

```diff
@@ -0,0 +1,41 @@
+URI specifications
+==================
+
+Authors: [Patrik Wallström](github.com/pawal)
+
+Reviewers:
+
+* * *
+
+# Abstract
+
+For different applications to handle ipfs references there need to be URIs registered. In order to for applications to understand how to handle the references, the URIs needs to have specificaitions. The generic syntax for URIs are described in [RFC3986](https://tools.ietf.org/html/rfc3986).
+
+The consensus of which URIs need to be specified can be found here: https://github.com/ipfs/go-ipfs/issues/1678#issuecomment-157478515
```

Comment:
I think this was updated and superseded by @lgierth's comment at https://github.com/ipfs/specs/pull/152#issuecomment-284628862, no?